tools/policy-gen/generator/pkg/parse/policyconfig.go

```diff
@@ -19,6 +19,7 @@ type PolicyConfig struct {
 	NameLower           string
 	Plural              string
 	SkipRegistration    bool
+	SkipDefault         bool
```

Comment:
I'd name the flag `SkipDefaultConf`, I think `SkipDefault` is a bit misleading

Reply:
Or maybe `SkipGetDefault`